.github/workflows/workflow.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+FILE_PATH="$1"
+
+# Arguments other than file path are passed to the command's arguments
+shift 1
+
+if [ ! -f "$FILE_PATH" ]; then
+    echo "No such file: $FILE_PATH"
+    exit 1
```

Comment:
If there is no special reason, change the indentation to 2 spaces wide to match the other languages in the repository.

Reply:
fixed in https://github.com/tier4/scenario_simulator_v2/pull/1223/commits/a35a58bd8e539793291e7eac702c4caae10d129e